src/main/java/gov/cms/ab2d/contracts/controller/HealthCheckController.java

```diff
@@ -0,0 +1,30 @@
+package gov.cms.ab2d.contracts.controller;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+@AllArgsConstructor
+@RestController
+@Slf4j
+public class HealthCheckController {
+
+    private final JdbcTemplate jdbcTemplate;
+
+    @GetMapping("/health")
+    public ResponseEntity<Void> getHealth() {
+        try {
+            jdbcTemplate.query("SELECT 1", x -> {
```

Comment:
I'm cool moving this to a service. As it was done as fast as possible, it's all KISS and no SOLID.